### ISSUE TITLE
Tenon: Colorbox buttons are empty 

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/express#1928]
## Context

Colorbox elements (container, buttons, etc) are added to every page, even if they are not being used.

## Tenon Error

 Error /  priority 100%
This button is empty.

(line: 1794, 0)

Web Content Accessibility Guidelines (WCAG) 2.0, Level AA: 2.4.6 Headings and Labels

This button has no text within it. People who use assistive technologies like voice-dictation software will not be able to refer to the button by name, and users of screen readers will not have any information about this button's purpose.

**Comment by kevincrafts** [comment]
Because the elements are hidden, this may not be an actual issue, but still reported by Tenon.

**Comment by kevincrafts** [comment]
JS library isn't loading correctly due to a missing version number.
